docs/blueprints/EP032.rst

```diff
@@ -0,0 +1,102 @@
+- Remove ``GenericFlow`` abstraction and its fields mappings since it's clear that OpenFlow1.3 with experimenter actions has been sufficient for production usage, and future efforts will focus on P4 and P4Runtime API. 
+- Bump v1 on endpoint routes and remove the ``GET flow/match/<dpid>`` endpoint, update the API spec accordingly. 
+- Adapt ``GET flow/stats/<dpid>`` to return the flows with their stats instead of matching.
+- Reassess if endpoints ``GET packet_count/sum/{{dpid}}`` and ``GET /bytes_count/sum/{{dpid}}`` are being valuable for important use cases or should be removed.
+
+mef_eline
+~~~~~~~~~
+
+- Update the consistency check to only run for EVCs that are enabled, inactive and haven't been updated within a time interval slightly greater than ``flow_manager.settings.CONSISTENCY_MIN_VERDICT_INTERVAL = 60 * 2``, and haven't got flows removed recently. Evaluate if the new time interval will be set on ``settings.DEPLOY_EVCS_INTERVAL`` or a new one and impact on existing adaptations on the existing ``WAIT_FOR_OLD_PATH``.
+- Subscribe to ``kytos/flow_manager.flow.removed`` and keep track when the EVC got flows removed by deriving its cookie, in case alien flows got removed then it'll be used to skip the consistency check.
+- Add a new DB ascending index on ``evcs.updated_at`` and implement a query filtering for EVCs where is less than or equal a given datetime ``"updated_at": {"$lte": dt}``
+- Update the consistency check to use the new ``PUT /traces`` that will be implemented for the EVCs that can run the consistency after every time interval.
+
+
+Issue #3
+========
+
+``flow_stats`` doesn't store statistics in a database yet, statistics values and their history will be lost if switches restart.
+
+Proposed solution
+-----------------
+
+Although it's desirable to move to MongoDB and have persistent FlowStats, there are still requirements and statistics use cases that need to be better understood to define a data model and API that will meet such requirements and integrate well with the front-end and other NApps. Replacing the existing in memory storage with new collection might not be the right move if later on time series queries might be needed, requiring more refactoring and breaking changes, so it's worth deferring this for a future opportunity where its priority is aligned with the major version objectives.
```

Comment:
When the time comes to implement database stats storage we've also got this issue here https://github.com/kytos-ng/kytos/issues/190, `kronos` has been structured to provide time series storage, if we'll leverage time series in the future we need to also check if we'll continue to maintain `kronos` or move an InfluxDB client to `flow_stats` in the future. That's far away but something to keep in mind, since the issue is linked on this comment we also don't lose track of it in he future.

Reply:
Fine.